src/ripple/app/main/Application.cpp

```diff
@@ -1033,7 +1021,9 @@ class ApplicationImp : public Application, public BasicApp
         {
             reportingETL_->stop();
 #ifdef RIPPLED_REPORTING
```

Comment:
I don't think you actually need the `ifdef` here. `RelationalDBInterfacePostgres` is not conditionally compiled. It's just `PgPool` that is conditionally compiled (along with the other classes in Pg.h).